services/QuillLMS/config/routes.rb

```diff
@@ -597,6 +597,17 @@
       end
     end
 
+    resources :districts do
+      collection do
+        post :search
+        get :search, to: 'districts#index'
+      end
+      member do
+        get :new_admin
+      end
+      resources :district_admins
```

Comment:
Based on the controller actions, this should probably be  `resources :district_admins, only: [:create, :destroy]`